src/core/playground/legacy/ed25519Signer.ts

```diff
@@ -0,0 +1,36 @@
+import { HexInput } from "../../../types";
+import { AccountAddress } from "../../accountAddress";
+import { LegacyAccountAuthenticatorEd25519 } from "../accountAuthenticator";
+import { Ed25519PrivateKey, Ed25519PublicKey } from "../ed25519";
+import type { BaseSigner } from "../interfaces";
+
+export class LegacyEd25519Signer implements BaseSigner<LegacyAccountAuthenticatorEd25519> {
```

Comment:
is this to replace Account?

Reply:
`LegacyEd25519Signer` and `Signer` will together replace `Account`

Reply:
I added the `Legacy` prefix, but open to use different names if we don't plan to deprecate them

Reply:
I think Legacy prefix is fine and good. We would need to support both forever